js/source/image_source.js

```diff
@@ -57,19 +56,27 @@ ImageSource.prototype = util.inherit(Evented, {
     onAdd: function(map) {
         this.map = map;
         if (this.image) {
-            this.createTile();
+            this.setCoordinates();
         }
     },
 
     /**
-     * Calculate which mercator tile is suitable for rendering the image in
-     * and create a buffer with the corner coordinates. These coordinates
-     * may be outside the tile, because raster tiles aren't clipped when rendering.
-     * @private
+     * Update image coordinates and rerender map
+     *
+     * @param {Array} coordinates lng, lat coordinates in order clockwise
```

Comment:
You should be a little more specific in your type declaration and description of this parameter. What do these coordinates represent? How many coordinates are expected? 

@jfirebaugh @ansis If we're going to make this method public, should we change the argument signature? We could

 - accept an array of 4 corner coordinates
 - accept an array of 2 corner coordinates
 - accept 4 corner coordinates as separate args
 - accept 2 corner coordinates as seperate args
 - accept an instance of `LngLatBounds`
 - (some combination of the above)